app/reports_generation/scripts/chart_generator.py

```diff
@@ -22,10 +30,46 @@ def __resolve_and_expand_user_path(path: Path) -> Path:
     return path.resolve().expanduser()
 
 
+def __get_all_default_actions():
+    full_actions_list = []
+    actions_data = read_json_file(DEFAULT_ACTIONS_PATH)
+    for app in APPS:
+        for test_type in TEST_TYPES:
+            for action in actions_data[app][test_type]:
+                full_actions_list.append(action)
+    return full_actions_list
+
+
+def __mark_as_app_specific(action_name: str):
```

Comment:
Confusion name and functionality of the method. Mark method should mark, but not decide what to mark.